test/e2e/snaps/test-snap-bip-32.spec.js

```diff
@@ -77,13 +80,16 @@ describe('Test Snap bip-32', function () {
           tag: 'button',
         });
 
-        // delay for npm installation
-        await driver.delay(2000);
-
         // switch back to test-snaps window
         windowHandles = await driver.waitUntilXWindowHandles(1, 1000, 10000);
         await driver.switchToWindowWithTitle('Test Snaps', windowHandles);
 
+        // wait for npm installation success
+        await driver.waitForSelector({
```

Comment:
I would like to discuss this with you as I ran all tests on both platforms over 10x today with zero failures, and i'm not sure how the `waitUntilXWindowHandles` would cause a failure here?